scripts/validate_docstrings.py

```diff
@@ -402,6 +402,12 @@ def examples_errors(self):
             error_msgs += f.getvalue()
         return error_msgs
 
+    @property
+    def examples_codes(self):
+        codes = doctest.DocTestParser().get_examples(self.raw_doc)
+        codes = [line.source for line in codes]
+        return ' '.join(codes)
```

Comment:
Can you return a list with each line instead? We'll reuse this property to validate pep8 too, and returning a single string with all the code joined is not useful in that case.

Also, at least to me `examples_codes` is somehow misleading. Can you use `examples_source_code` instead?